go.mod

```diff
@@ -1,8 +1,9 @@
-module github.com/adyen/adyen-go-api-library
+module github.com/adyen/adyen-go-api-library/v2
```

Comment:
I thought we were not gonna do this. I don't see any value in providing support for v1 & v2 for this change and given the lib is new, I would rather just do a clean release with just a major version bump

Reply:
if someone wants to use v1 they can always use that particular tag